example/algorithms_evaluation/fixed_points_new.py

```
@@ -0,0 +1,12 @@
+import sys
+
+from biodivine_aeon import *
+
+bn = BooleanNetwork.from_file(sys.argv[1])
+bn = bn.infer_valid_graph()
+
+fixed_points = FixedPointsComp.create_from(bn).symbolic()
```

Ruff (0.12.2)

`FixedPointsComp` may be undefined, or defined from star imports

(F405)